addons/mail/static/src/thread/message.js

```diff
@@ -4,11 +4,12 @@ import { useMessaging } from "../messaging_hook";
 import { RelativeTime } from "./relative_time";
 import { Component } from "@odoo/owl";
 import { useService } from "@web/core/utils/hooks";
+import { session } from '@web/session';
```

Comment:
don't import the session directly. it's deprecated. you can use the `isAdmin` key from user service